data.sql

```diff
@@ -76,3 +76,38 @@ VALUES	(1,1,1,'2020-05-24'),
 			(18,9,2,'2020-08-03'),
 			(19,10,3,'2020-05-24'),
 			(20,10,1, '2021-01-11');
+
+
+INSERT INTO specializations(vets_id, species_id) VALUES
+((SELECT id from vets where name = 'William Tatcher'),(SELECT id from species where name = 'Pokemon')),
+((SELECT id from vets where name = 'Stephanie Mendez'),(SELECT id from species where name = 'Digimon')),
+((SELECT id from vets where name = 'Stephanie Mendez'),(SELECT id from species where name = 'Pokemon')),
+((SELECT id from vets where name = 'Jack Harkness'),(SELECT id from species where name = 'Digimon'));
+
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (1, 1, '2020-05-24');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (1, 3, '2020-07-22');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (2, 4, '2021-02-02');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (3, 2, '2020-01-05');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (3, 2, '2020-03-08');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (3, 2, '2020-05-14');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (4, 3, '2021-05-04');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (5, 4, '2021-02-24');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (6, 2, '2019-12-21');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (6, 1, '2020-08-10');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (6, 2, '2021-04-07');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (7, 3, '2019-09-29');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (8, 4, '2020-10-03');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (8, 4, '2020-11-04');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (9, 2, '2019-01-24');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (9, 2, '2019-05-15');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (9, 2, '2020-02-27');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (9, 2, '2020-08-03');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (10, 3, '2020-05-24');
+INSERT INTO visits (animal_id, vets_id, visit_date ) VALUES (10, 1, '2021-01-11');
+
+
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vets_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
- [ ] Your BEFORE execution times for the first and third queries are significantly lower than 1000ms. I suggest running the populate database query a few times more as described in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.)